lib/ioki/model/operator/passenger_option.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+module Ioki
+  module Model
+    module Operator
+      class PassengerOption < Base
+        attribute :type,
+                  on:   :read,
+                  type: :string
+
+        attribute :id,
+                  on:   :read,
+                  type: :string
+
+        attribute :created_at,
+                  on:   :read,
+                  type: :date_time
+
+        attribute :updated_at,
+                  on:   :read,
+                  type: :date_time
+
+        attribute :slug,
+                  on:   [:read, :create],
+                  type: :string
+
+        attribute :default_value,
+                  on:   [:read, :create, :update],
+                  type: [:string, :boolean, :integer]
```

Comment:
Did you test whether this deserializes integers/booleans correctly? I faintly remember that we had to put `string` at the end, because we start deserializing at the beginning and stop when it works. I believe deserializing as string always works. But I'm not sure if it's implemented as I describe it.